scripts/sanitycheck

```diff
@@ -2754,19 +2767,37 @@ def main():
         export_tests(options.export_tests, tests)
         return
 
+    run_individual_tests = []
+
+    if options.test:
+        run_individual_tests = options.test
 
-    if options.list_tests:
+    if options.list_tests or options.sub_test:
         cnt = 0
         unq = []
+        run_individual_tests = []
         for n,tc in ts.testcases.items():
             for c in tc.cases:
+                if options.sub_test and c in options.sub_test:
+                    if tc.name not in run_individual_tests:
+                        run_individual_tests.append(tc.name)
                 unq.append(c)
 
-        for u in sorted(set(unq)):
-            cnt = cnt + 1
-            print(" - {}".format(u))
-        print("{} total.".format(cnt))
-        return
+        if options.sub_test:
+            if run_individual_tests:
+                info("Running the following tests:")
+                for t in run_individual_tests:
```

Comment:
Why a for loop? Is it for the future, when `run_individual_tests` could be a multi-element list?
For now, I get only a maximum of one string in this variable when I run sanitycheck with the sub-test name. Am I missing something here?